src/Symfony/Component/Asset/Tests/Context/RequestStackContextTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\Tests\Context;
+
+use Symfony\Component\Asset\Context\RequestStackContext;
+
+class RequestStackContextTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConstructor()
+    {
+        $requestStack = $this->getMockBuilder('Symfony\Component\HttpFoundation\RequestStack')
+            ->getMock();
+        $requestStackContext = new RequestStackContext($requestStack);
+
+        $this->assertInstanceOf('Symfony\Component\Asset\Context\RequestStackContext', $requestStackContext);
+    }
+
+    public function testGetBasePath()
+    {
+        $requestStack = $this->getMockBuilder('Symfony\Component\HttpFoundation\RequestStack')
+            ->getMock();
+        $requestStackContext = new RequestStackContext($requestStack);
+
+        $this->assertEmpty($requestStackContext->getBasePath());
```

Comment:
This should be the end of the test case. As a rule of thumb it's good to follow an arrange-act-assert flow (for readability and maintainability). I know there's lots of tests violating this rule in Symfony, but I'd rather don't add more of them.

This also shows a problem with naming test cases after a method being tested, as it basically forces you to put everything in a single method. I prefer to summarise the requirement/rule (for example: testItIsInitialisedWithAnEmptyBasePath, testItRetrievesTheBasePathFromTheMasterRequest).